de-lib/src/main/java/org/iplantc/de/apps/client/views/details/AppDetailsViewImpl.java

```diff
@@ -246,6 +249,38 @@ public void onClick(ClickEvent event) {
         } else {
             helpLink.setVisible(false);
         }
+
+        // React integration
+//        Scheduler.get().scheduleFinally(new Scheduler.ScheduledCommand() {
+//
+//            @Override
+//            public void execute() {
+//                final Splittable appJson = AutoBeanCodex.encode(AutoBeanUtils.getAutoBean(app));
+//
+//                ReactToolDetails.ToolDetailProps detailProps = new ReactToolDetails.ToolDetailProps();
+//
+//                detailProps.appearance = appearance;
+//                detailProps.app = appJson;
+//
+//                CyVerseReactComponents.render(ReactToolDetails.ToolDetails, detailProps, DivElement.as(toolsContainer.getElement()));
+//
+//                ReactCategoryTree.CategoryTreeProps treeProps = new ReactCategoryTree.CategoryTreeProps();
+//                treeProps.app = appJson;
+//                treeProps.appearance = appearance;
+//                treeProps.presenter = AppDetailsViewImpl.this;
+//
+//                DivElement hierarchyElem = DivElement.as(hierarchyWidget.getElement());
+//                CyVerseReactComponents.render(ReactCategoryTree.CategoryTree, treeProps, hierarchyElem);
+//            }
+//        });
```

Comment:
I'm not completely sold on `rc-tree`, and I'm not sure if we wanted to switch the tool details from accordion panels to `cyverse-ui` MediaCards yet.

If everyone thinks they look OK, then I'll at least add back the details UI and remove the old GXT version.

Reply:
Ya lets find out if there a better tree component out there. Here is the list of treeviews I found - https://react.rocks/tag/TreeView

Reply:
https://elbywan.github.io/bosket/react/index.html#Presets#ExplorerView#Basic%20Usage looks pretty good too, I'm just not sure yet if it will support remote/lazy loading or how hard it would be to style like `cyverse-ui`.

Reply:
Looks like it will...from the link

- Drag and drop, with a custom drag image.
- Asynchronous children loading
- Multiselection using keyboard modifiers (shift / ctrl or cmd)
- Transitions on fold / unfold
- Alphabetical sort
- Search bar

Reply:
Looks like I ran out of time to cleanup the tool details React view, so I'll just leave it commented out for now, and hopefully I'll find some time after my vacation to finish it up and remove the old GWT view.